lib/3llo/controller.rb

```diff
@@ -19,7 +19,12 @@ def start(init_command)
       end
 
       loop do
-        command_buffer = Readline.readline("\e[15;48;5;27m 3llo \e[0m > ", true)
+        begin
+          board = Application.fetch_board!()
+          command_buffer = Readline.readline("\e[15;48;5;27m 3llo \e[0m > \e[1m\e[33m #{board.name} \e[0m > ", true)
```

Comment:
I wasn't sure what `\e[15;48;5;27m` and `\e[0m` meant. After doing some research I found out that they are for colorizing the output. But when I try it on my terminal the color doesn't change at all. Is this dependent on the terminal for the output to show color?

Reply:
What's your expectation for the Readline output?

Reply:
I thought it would show colors like how the it's shown in the README GIF
![Screen Shot 2020-05-12 at 12 57 52 AM](https://user-images.githubusercontent.com/19693587/81640132-bfa3c480-93eb-11ea-9b83-afe54aa431b1.png)

Reply:
I recommend use this background color instead for easy reading.

Reply:
<img width="241" alt="Screen Shot 2020-05-12 at 12 10 18" src="https://user-images.githubusercontent.com/17459154/81640865-d973f480-9449-11ea-926a-fba46712a4a2.png">